modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastsViewModel.kt

```diff
@@ -310,7 +310,7 @@ class PodcastsViewModel
     fun refreshSuggestedFolders() {
         viewModelScope.launch {
             if (FeatureFlag.isEnabled(Feature.SUGGESTED_FOLDERS)) {
-                val uuids = podcastManager.findSubscribedUuids()
+                val uuids = podcastManager.findFollowedPodcastsNotInFolderUuid()
```

Comment:
It would be good to cache this so it doesn't call the server each time the page loads.